text/006-bft-based-ordering-service.md

```diff
@@ -0,0 +1,404 @@
+A [shorter version](https://arxiv.org/abs/2107.06922) of the paper appears in the [IEEE International Conference on Blockchain and Cryptocurrency 2021](https://icbc2021.ieee-icbc.org/program.1.html)
+
+|                              LAN                              |                                                                                                                             WAN                                                                                                                            |
+|:-------------------------------------------------------------:|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|  ![alt text](https://raw.githubusercontent.com/SmartBFT-Go/SmartBFT-Go.github.io/master/lan.png)                            |                                                                                                                           ![alt text](https://raw.githubusercontent.com/SmartBFT-Go/SmartBFT-Go.github.io/master/wan.png)                                                                                                                          |
+| In LAN, the throughput is above 2000 transactions per second. It can be observed that even smaller batch sizes do not significantly degrede the performance of the system. Furthermore, the size of the cluster also didn't affect the throughput significantly.| In WAN, nodes were deployed all over the world: Dallas, London, Washington, San Jose, Toronto, Sydney, Milan, Chennai, Hong Kong and Tokyo and with a decent batch size (1000 transactions per block) a throughput of 1,000 transactions per block was achieved. |
+
+# Possible questions and their answers
+
+-  **Q**: Why SmartBFT? There are more performant consensus protocols out there. SmartBFT is essentially a non pipelined PBFT, which isn't great for performance.  
+
+   **A**: By following the approaches explained in the previous [two](https://github.com/hyperledger/fabric-rfcs/pull/48), [RFCs ](https://github.com/hyperledger/fabric-rfcs/pull/50) in the series, integrating SmartBFT into Fabric does not block any other protocol to be integrated as well.  As a matter of fact, it only makes integration of other Byzantine Fault Tolerant libraries easier, as it will surely create re-usable code.
+
+-  **Q**: Who authored and maintains the SmartBFT library why should we trust it? 
+
+   **A**: The SmartBFT consensus library was authored by [a group of people](https://github.com/SmartBFT-Go/consensus/graphs/contributors) from IBM Research, Israel. All of them being contributors to Fabric core, and some of them being maintainers. In mid 2021 its maintenance was transferred to [Ideas](https://scientificideas.org), the Interdisciplinary Blockchain Research and Expertise center. 
```

Comment:
Why was it 'transferred' to Ideas? Seems it would make sense to jointly maintain it between the original authors AND Ideas.